chia/timelord/timelord_launcher.py

```diff
@@ -39,7 +39,7 @@ def find_vdf_client() -> pathlib.Path:
     raise FileNotFoundError("can't find vdf_client binary")
 
 
-async def spawn_process(host: str, port: int, counter: int, lock: asyncio.Lock, prefer_ipv6: Optional[bool]):
+async def spawn_process(host: str, port: int, counter: int, lock: asyncio.Lock, prefer_ipv6: bool):
```

Comment:
would this break a lot of call sites?

Reply:
Not more than i anyway already touched with this PR, so i also changed it in [d581b92](https://github.com/Chia-Network/chia-blockchain/pull/14068/commits/d581b9277fba2dbf7bed1ec56fb8d2e06024994d).